packages/web-ui/src/ds/molecules/Chat/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { CompileError, ContentType } from '@latitude-data/compiler'
+import { Text } from '$ui/ds/atoms'
+
+import { Message } from '../Message'
+
+export function ErrorMessage({ error }: { error: Error }) {
+  return (
+    <div className='flex flex-col gap-2'>
+      <Message
+        role='Error'
+        content={[{ type: ContentType.text, value: error.message }]}
+        badgeVariant='destructive'
+        textColor='destructive'
+      />
+      {error instanceof CompileError && (
```

Comment:
shouldn't we do something if it's something else? Can it ever be something else?

Reply:
It could be something else. For any error type (including `CompileError`), the error message is rendered. Additionally, if it's a `CompileError`, we also render the error context, which is what is happening here.

![image](https://github.com/user-attachments/assets/6b5089ef-428d-4b5a-b40d-32d2bd4ae502)